### ISSUE TITLE
Does the community consider supporting multiple remote url cache?

### ISSUE THREAD

**Issue by justadogistaken** [distribution/distribution#3423]
Allow to describe the scene first:
the registry has multiple remote url:
https://reg.test1.com
https://reg.test2.com

when I do docker pull reg.test1.com/library/nginx:latest
the registry will go to reg.test1.com pull the image, and serve the image for docker client, in the meanwhile, cache the image.
I know this feature also needs some changes from docker. But will the community support such feature?

**Comment by deleteriousEffect** [comment]
Hi @justadogistaken, thanks for submitting this issue!

Just to clarify, you would like to have a single instance of the registry running [as a pull-through cache](https://docs.docker.com/registry/recipes/mirror/) for multiple remote registries at the same time?

**Comment by justadogistaken** [comment]
Yes, this is what I want to do. And I hacked the distribution to support this feature, but it also needs some changes from docekr. Thus I want to know if the community plans to support the feature.

**Comment by deleteriousEffect** [comment]
@justadogistaken Could you open the docker-side issue here: https://github.com/moby/moby and provide a link here?

As far as community plans to support the feature, I don't oppose it, if there's interest and time it's worth an investigation.

**Comment by justadogistaken** [comment]
@deleteriousEffect ok, I am to file a issue there
